lms/product/blackboard/__init__.py

```diff
@@ -13,6 +12,3 @@ def includeme(config):  # pragma: nocover
     config.register_service_factory(
         BlackboardCourseCopyPlugin.factory, iface=BlackboardCourseCopyPlugin
     )
-    config.register_service_factory(
-        BlackboardMiscPlugin.factory, iface=BlackboardMiscPlugin
```

Comment:
No need for this plugin, no other differences than the grading comments being disabled, which is removed here.